packages/mdc-menu/_mixins.scss

```diff
@@ -0,0 +1,25 @@
+//
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+@import "./variables";
+
+@mixin mdc-menu-width($width) {
+  @if unitless($width) {
+    width: $width * $mdc-menu-width-base;
```

Comment:
I have some questions about this mixin...

1. What is the motive behind supporting passing a unitless multiplier? I didn't seem to see anything in the spec suggesting this...
2. What is the motive for using this mixin at all, rather than letting the menu auto-size based on its content?

Reply:
Spec guidelines recommend the menu width being a multiple of `56px` (min: 2, max: 6). I wanted to have a mixin to make the multiple apparent while also allowing the user to override it with their custom value.